static/app/components/nav/overlay.tsx

```diff
@@ -0,0 +1,60 @@
+import {type ComponentProps, forwardRef} from 'react';
+import styled from '@emotion/styled';
+
+import InteractionStateLayer from 'sentry/components/interactionStateLayer';
+import Link from 'sentry/components/links/link';
+import {Overlay} from 'sentry/components/overlay';
+import {space} from 'sentry/styles/space';
+
+export const OverlayMenu = forwardRef<
+  HTMLDivElement,
+  Omit<ComponentProps<typeof StyledOverlay>, `on${string}`>
+>(({children, ...props}, ref) => {
+  return (
+    <StyledOverlay {...props} ref={ref}>
+      {children}
+    </StyledOverlay>
+  );
+});
+
+export const OverlayMenuLink = forwardRef<HTMLAnchorElement, ComponentProps<typeof Link>>(
```

Comment:
Are these going to be used in a followup pr?

Reply:
This file was from an earlier version of the PR! Removed in f1efc6c6948